### DISCUSSION TITLE
Permission denied while trying to run Docker deamon

### DISCUSSION THREAD

**Discussion by arujjval** [nektos/act#2404] [Q&A]
while running act with docker running, I am getting this error 

`Error: permission denied while trying to connect to the Docker daemon socket at unix:///var/run/docker.sock: Post "http://%2Fvar%2Frun%2Fdocker.sock/v1.45/images/create?fromImage=node&tag=16-buster-slim": dial unix /var/run/docker.sock: connect: permission denied
`

I have all the necessary permissions to run docker and it is working for other tasks. How to resolve this?

**Comment by bkaradzic** [comment]
This solves the issue:
https://community.bigbeartechworld.com/t/how-to-fix-docker-daemon-permission-issues-on-linux/2103
